doc/ocaml-ecosystem.rst

```diff
@@ -0,0 +1,89 @@
+The OCaml Ecosystem
+===================
+
+The OCaml ecosystem is not monolithic: the compiler and tools are not
+maintained by the same entities. As such, it can be difficult to understand the
+history and roles of the various pieces of this ecosystem. The goal of this
+page is to give a quick overview of the situation and of the role that Dune
+plays in it.
+
+The Compiler Distribution: Compiling and Linking
```

Comment:
```suggestion
The OCaml Compiler Distribution: Compiling and Linking
```
also the underline

Reply:
FYI, you can select both lines and make a multiline suggestion